gce-jetty/src/main/docker/Dockerfile

```diff
@@ -27,11 +27,14 @@ RUN mv /usr/local/jetty-distribution-* $JETTY_HOME && chown -R jetty:jetty $JETT
 ENV JETTY_BASE /var/lib/jetty
 RUN mkdir -p $JETTY_BASE
 WORKDIR $JETTY_BASE
-RUN java -jar $JETTY_HOME/start.jar --add-to-startd=http,deploy,setuid && \
+RUN java -jar $JETTY_HOME/start.jar --add-to-startd=http,deploy,jsp,jstl,setuid && \
     chown -R jetty:jetty $JETTY_BASE && \
-    mkdir /tmp/jetty
+    mkdir /tmp/jetty && chown -R jetty:jetty /tmp/jetty
+
+RUN ln -s $JETTY_BASE/webapps/root /app
+WORKDIR /app
```

Comment:
Not also that the official jetty-docker image now uses a small shell script as the entry point so that it is easy to run with a bash shell.  I think it is worthwhile to mimic that approach as it will give us better doco links.

For GAE it is not a problem as we introduce our own start script anyway!